Xrpl.C/Ripple.Binary.Codec/Util/Utils.cs

```diff
@@ -686,27 +550,20 @@ public static uint ToUInt32(byte[] value, int startIndex)
         ///   System.ArgumentOutOfRangeException:
         ///     startIndex is less than zero or greater than the length of value minus 1.
         ///</summary>
-        public static ulong ToUInt64(byte[] value, int startIndex)
-        {
-            if (IsLittleEndian)
-            {
-                return BitConverter.ToUInt64(value, startIndex);
-            }
-            return BitConverter.ToUInt64(value.Reverse().ToArray(), value.Length - sizeof(UInt64) - startIndex);
-        }
+        public static ulong ToUInt64(byte[] value, int startIndex) => IsLittleEndian ? BitConverter.ToUInt64(value, startIndex) : BitConverter.ToUInt64(value.Reverse().ToArray(), value.Length - sizeof(UInt64) - startIndex);
     }
     public static class ChannelUtils
     {
         public static byte[] EncodeChannel(string channelHex, long amount)
         {
             byte[] HASH_CHANNEL_SIGN = { 0x43, 0x4C, 0x4D, 0x00 };
-            byte[] AMOUNT_HEX_ARRAY = Bits.GetBytes(amount);
-            byte[] CHANNEL_HEX_ARRAY = Convert.FromHexString(channelHex);
+            var AMOUNT_HEX_ARRAY = Bits.GetBytes(amount);
```

Comment:
Is this the IDE you're using or are you removing the type definition? I like byte[] VARIABLE much more than a dynamic `var`. 

Can you explain why you chose to remove the type definitions?

Reply:
best description of why I use 'var' instead of strict type definition - [here](http://resharper.blogspot.com/2008/03/varification-using-implicitly-typed.html)